sw/device/tests/BUILD

```diff
@@ -1782,6 +1782,28 @@ opentitan_functest(
     ],
 )
 
+opentitan_functest(
+    name = "i2c_host_ambient_light_detector_test",
+    srcs = ["i2c_host_ambient_light_detector_test.c"],
+    cw310 = cw310_params(
+        tags = ["skip_in_ci"],  # Requires the BoB in CI.
```

Comment:
```suggestion
        tags = ["manual"],  # Requires the BoB in CI.
```
or run it in CI?

Reply:
Marked this (and the others) as manual.

I'm not sure I 100% understand what "skip_in_ci" does - is it to skip for PR runs but not nightlies (https://github.com/lowRISC/opentitan/pull/18676#discussion_r1199496053)? Is there any docs for it?

Reply:
The general differences between "skip_in_ci" and "manual" are:
1. "skip_in_ci" is a custom tag that allows us to filter out those tests in CI (with a `--test_tag_filters=-skip_in_ci` flag) but still enables running the test locally if you run `bazel test //...`
2. "manual" will skip running in CI and running locally since it does not get run with `bazel test //...`, and is a special pre-existing Bazel feature
3. "skip_in_ci" was a custom tag added during the ROM E2E test development sprint. Many of these tests run across all LC states in a loop, and made CI ROM E2E test job runtimes blow up because the bitstreams are reloaded between test runs. To prevent this blow up in runtimes, the filtering by tag mechanism was developed. We still run all tests nightly though as you have observed.

In this case you don't want the test to run unless it is "manually" invoked, i.e., NOT through a `bazel test //...` but through a `bazel test //<package>:<label>` invocation.